src/modules/major/dto/major-relation-field-request-query.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { StringBooleanTransform } from '@src/common/common';
+import { Transform } from 'class-transformer';
+import { IsBoolean, IsOptional } from 'class-validator';
+
+export class MajorRelationFieldRequestQueryDto {
+  @ApiPropertyOptional({
+    description: 'mainSKills 를 true 로 주면 response 의 majors 가 추가됩니다.',
+    default: false,
+    enum: ['true', 'false'],
```

Comment:
enum.ts에 만들어 놓으신 BooleanString 사용하지 않으신 이유가 있을까요?


```suggestion
    enum: [BooleanString.True, BooleanString.False] 
```

Reply:
저게 처음에 enum 사용했다가 머가 이상해서 바꾼걸로 기억하는데 한번 더 확인해보고 코멘트에 남기도록 할게여